src/apps/organisations/redux/actions.ts

```diff
@@ -4,19 +4,48 @@ import {
   startAction, 
   completeAction,
   progressAction,
-  resetAction
+  resetAction,
+  indexedAction
  } from "../../../redux";
 import {
-  GET_USER_ORGS_ACTION,
-  CREATE_ORGANISATION_ACTION
+  CREATE_ORGANISATION_ACTION,
+  EDIT_ORGANISATION_ACTION,
+  GET_ORG_ACTION,
+  GET_ORG_COLLECTIONS_ACTION,
+  GET_ORG_SOURCES_ACTION,
+  DELETE_ORGANISATION_ACTION,
+  RETRIEVE_ORGS_ACTION
 } from "./actionTypes";
-import { Organisation } from '../types';
+import {PERSONAL_ORGS_ACTION_INDEX} from "./constants";
+import { Organisation, EditableOrganisationFields } from '../types';
 
 const createOrgsAction = createActionThunk(
   CREATE_ORGANISATION_ACTION,
   api.create
 );
 
+const editOrgsAction = createActionThunk(
```

Comment:
One thing I don't really understand here is why we have both this `editOrgsAction` and `editOrganisationAction` below. Do we actually need these? (I'd tend to prefer the versions using `createActionThunk` because that hooks in our general error handling).

Reply:
I guess I made a repetition there, I will clear that.